content/homebrew-build-provenance.md

```diff
@@ -0,0 +1,54 @@
++++
+title = "Homebrew's Sigstore-powered provenance is in beta"
+date = "2024-05-16"
+tags = ["sigstore", "security", "homebrew"]
+draft = false
+author = "William Woodruff (Trail of Bits), Hayden Blauzvern (Google)"
+type = "post"
++++
+
+Last November, [Alpha-Omega] and [Trail of Bits] announced a
+[collaboration to bring build provenance to homebrew-core].
```

Comment:
Can we link to https://openssf.org/blog/2023/11/06/alpha-omega-grant-to-help-homebrew-reach-slsa-build-level-2/ also?

Reply:
Yep, I'll work that in 🙂